src/bosh-softlayer-cpi/softlayer/common/agent_options.go

```diff
@@ -42,6 +42,14 @@ func (o AgentOptions) Validate() error {
 	return nil
 }
 
+func (o *AgentOptions) GetVcapPassword() string {
+	return o.vcapPassword
+}
+
+func (o *AgentOptions) SetVcapPassword(vcapPassword string) {
```

Comment:
Let's discuss tonight and decide right then and there. Seems that this is taking way more time than you or I want.